src/Microsoft.AspNet.SignalR.Core/Messaging/MessageBus.cs

```diff
@@ -258,16 +258,18 @@ protected void ScheduleEvent(string eventKey)
             Topic topic;
             if (Topics.TryGetValue(eventKey, out topic))
             {
-                ScheduleTopic(topic);
+                ScheduleTopic(eventKey, topic);
             }
         }
 
-        private void ScheduleTopic(Topic topic)
+        private void ScheduleTopic(string eventKey, Topic topic)
         {
             try
             {
                 topic.SubscriptionLock.EnterReadLock();
 
+                _trace.TraceVerbose("Scheduling topic '{0}'. Number of subscriptions: {1}", eventKey, topic.Subscriptions.Count);
```

Comment:
It tells me how many subscriptions subscribed to a given topic. Given that I am not exactly sure what I am looking for it might not be that useless. I would prefer to keep it.